src/connectors/breadcrumb/connectBreadcrumb.js

```diff
@@ -56,20 +44,25 @@ Full documentation available at https://community.algolia.com/instantsearch.js/v
  * @return {function(CustomBreadcrumbWidgetOptions)} Re-usable widget factory for a custom **Breadcrumb** widget.
  */
 export default function connectBreadcrumb(renderFn, unmountFn) {
-  checkRendering(renderFn, usage);
+  checkRendering(renderFn, withUsage());
   return (widgetParams = {}) => {
     const {
       attributes,
       separator = ' > ',
       rootPath = null,
       transformItems = items => items,
     } = widgetParams;
-    const [hierarchicalFacetName] = attributes;
```

Comment:
This line used to fail when `attributes` was not an array. I moved it after the check and added a test.